src/js/api/config.js

```diff
@@ -126,6 +126,8 @@ define([
             config.playlist = [ obj ];
         }
 
+        config.qualityLabels = config.qualityLabels || config.hlslabels;
```

Comment:
We can verify with @esboyd, but I believe all config names you be entirely lowercase `qualitylabels`.